flink-java/src/test/java/org/apache/flink/api/java/operator/SortPartitionTest.java

```diff
@@ -169,6 +169,38 @@ public void testSortPartitionWithExpressionKeys4() {
 		tupleDs.sortPartition("f3", Order.ASCENDING);
 	}
 
+	@Test
```

Comment:
I suggest add more tests:
- mixing `SelectorFunctionKeys` and `ExpressionKeys` -> Should not work
- use a `KeySelector` that returns a composite type, e.g., a `Tuple3`